packages/larva-css/src/03-utilities/u-text-decoration.common.inline.scss

```diff
@@ -0,0 +1,6 @@
+@import '@penskemediacorp/larva-scss/dist/tools';
+
+.lrv-u-text-decoration-underline\:hover:hover {
```

Comment:
What is the purpose of the \ and having the pseudo selector twice?

Reply:
The first `\:hover` is escaped so that it can be written `lrv-u-text-decoration-underline:hover` - the second one is the actual pseudo selector.